azure_arc_k8s_jumpstart/docs/local_kind_gitops_helm.md

```diff
@@ -0,0 +1,198 @@
+# Manually setting up an ingress contoller on kind
+
+The demo application that will be deployed later in this guide relies on an ingress controller. For ingress controllers to work on kind, a specific configuration of the ingress needs to be deployed. For more information related to this, please refer to the [kind documentation](https://kind.sigs.k8s.io/docs/user/ingress/).
+
+## Deployment
+* Run the following command to install the nginx ingress controller on kind:
+    ```bash
+    kubectl apply -f https://raw.githubusercontent.com/kubernetes/ingress-nginx/master/deploy/static/provider/kind/deploy.yaml
+    ```
+* This command will create a new namespace and deploy the required components in this namespace. To verify the deployment of the ingress controller was successful, make sure the pod with name `ingress-nginx-controller-<random id>-<random id>` is in a running state with 1/1 containers ready:
+
+![](../img/local_kind_gitops_helm/03.png)
+
+* Finally, test that the ingress is responding to traffic. To test this, either browse to [http://localhost](http://localhost) or use the command line to connect to `localhost`. You should get a HTTP 404 reponse with a nginx footer. This shows that the ingress is working. The 404 response is to be expected since you haven't setup an ingress route yet. You will do that in the next section.
+
+![](../img/local_kind_gitops_helm/04.png)
+![](../img/local_kind_gitops_helm/05.png)
+
+
+
+# Cluster-level Config vs. Namespace-level Config 
+
+## Cluster-level Config
+
+With Cluster-level GitOps config, the goal is to have "horizontal components" or "management components" deployed on your Kubernetes cluster which will then be used by your applications. Good examples are Service Meshes, Security products, Monitoring solutions, etc. You will not be creating a cluster-level config in this guide. For an example of a cluster-level configuration please refer to the [helm based gitops on AKS scenario](aks_gitops_helm.md).
+
+## Namespace-level Config
+
+With Namespace-level GitOps config, the goal is to have Kubernetes resources deployed only in the namespace selected. The most obvious use-case here is simply your application and its respective pods, services, ingress routes, etc. In the next section will have the "Hello Arc" application deployed on a dedicated namespace. 
+
+# Azure Arc Kubernetes GitOps Configuration with Helm
+
+## The Mechanism (In a nutshell)
+
+In the process of creating Azure Arc GitOps configuration, [Weaveworks Flux Kubernetes Operator](https://github.com/fluxcd/flux) is deployed on the cluster. 
+
+The Operator is aware to an "HelmRelease" Custom Resource Definition (CRD). This HelmRelease points to a git repo and can optionally contain specific values to input into the helm chart. Due to this configuration, a user can choose to leave the chart values intact or to have different values for different releases. 
```

Comment:
It is actually the flux-helm operator that is looking at the Helm release so you should point that out.

Reply:
I didn't adapt that piece, didn't really know how to reword. I'll take suggestions here, and if we update this statement - we might want to changes this across the different scenarios in the repo.